tools/github/release/manager.py

```diff
@@ -0,0 +1,143 @@
+import pathlib
+import re
+from functools import cached_property
+from typing import Dict, List, Optional, Pattern, Type, Union
+
+import verboselogs  # type:ignore
+
+import packaging.version
+
+import aiohttp
+
+import gidgethub.abc
+import gidgethub.aiohttp
+
+from tools.base import abstract
+from tools.base.functional import async_property
+
+from tools.github.release.abstract import AGithubRelease, AGithubReleaseManager
+from tools.github.release.exceptions import GithubReleaseError
+# from tools.github.release.release import GithubRelease
```

Comment:
?

Reply:
in order to land them separately i commented out the interdependent bits in the tests and class